app/lib/constants/site_config.rb

```diff
@@ -97,6 +97,10 @@ module SiteConfig
         description: "Used as the site favicon",
         placeholder: "https://image.url"
       },
+      feed_strategy: {
+        description: "Determines the main feed algorithm approach the app takes (basic or optimized)",
```

Comment:
This message might be confusing in the admin UI since people have no idea what basic and optimized mean. If you give me these options I'd definitely pick optimized without further info. 😉 

![image](https://user-images.githubusercontent.com/47985/92681931-39779a80-f359-11ea-86e7-2887b8e6c631.png)